src/components/header/Header.svelte

```diff
@@ -35,7 +41,13 @@
         <Link to="/rewards" class="link">REWARDS</Link>
       </li>
       <li class:active={false}>
-        <a href={aboutURL} target="_blank" class="link">ABOUT</a>
+        <a
+          href={$oceanUnlockDate && $oceanUnlockDate > moment()
+            ? "https://blog.oceanprotocol.com/ocean-data-farming-series-c7922f1d0e45"
+            : "https://docs.oceanprotocol.com/veocean-data-farming-tutorial"}
```

Comment:
I don't think we need to be this granular with the about link.

Let's just improve the docs and update the about link to point to it.

Reply:
done